### ISSUE TITLE
Not working in Microsoft Edge

### ISSUE THREAD

**Issue by anselmh** [anselmh/object-fit#45]
Due to a bug in `getMatchedCSSRules` in Microsoft Edge this polyfill won’t work as expected in Microsoft Edge. As this function is basically the heart of the polyfill, I have no clue how to fix this without refactoring the whole polyfill (which due to other restrictions is unwanted).

I’m very sorry about this. In case anyone has a solution, please let me know or send a Pull Request.

**Comment by julmot** [comment]
Wontfix for Edge?

**Comment by anselmh** [comment]
Yes, I have no way to fix this currently as Edge does not provide any alternative and therefore the aimed solution is not usable anymore. I’m sorry about this and have been poking people at Microsoft regarding this and the native support but did not get an answer. :(
